lib/ontologies_linked_data/models/mod/semantic_artefact.rb

```diff
@@ -192,9 +192,9 @@ def bring(*attributes)
 
             def self.all_artefacts(options = {})
                 onts = if options[:also_include_views]
-                        Ontology.where.to_a
+                        Ontology.where(viewingRestriction: 'public').to_a
                     else
-                        Ontology.where.filter(Goo::Filter.new(:viewOf).unbound).include(:acronym).to_a
+                        Ontology.where(viewingRestriction: 'public').filter(Goo::Filter.new(:viewOf).unbound).include(:acronym).to_a
```

Comment:
when you admin you can see them all.
And if you are the admin of the ontology you can see it also even if private.